### PR TITLE
Implements attribute tests closes #13

### LINKED ISSUE #13: Make a Hero.attribute_probe() to test for attributes (Probe auf Attribute)

### PR DESCRIPTION
**Pull request by arnohakk**

### CODE REVIEW

File: probemaker.py [hunk 1]
```diff
@@ -133,7 +134,35 @@ def __init__(self, file, show_values=False):
...
+        elif roll != 20:
+            print(f"{self.name} has failed")
+
+        if roll == 20:
+            print(f"{self.name} has failed, will it be a complete disaster?")
+            roll2 = randint(1, 20)
+            res2 = self.attr[attr] - roll2 + mod
+            if res <= 0:
+                print("Yes....")
+            else:
```
Comment (casibus) [inline probemaker.py:162]: maybe this method should return the actual result value (useful for logging/storing later)

File: probemaker.py [hunk 2]
```diff
@@ -213,7 +242,7 @@ def probe(self, talent: str, mod: int=0):
         elif mega_patz:
             print(self.name + ' is an idiot and mega patzed.')
```
Comment (casibus) [inline probemaker.py:243]: This method could return the result as well

**Review by casibus — COMMENTED** [review]
und wieder ein feature! 👍

File: probemaker.py (hunk 2 above)
Comment (arnohakk) [reply probemaker.py:243]: Let take care of this once we start logging, I also think we should do something about the general structure of the code

File: probemaker.py (hunk 1 above)
Comment (arnohakk) [reply probemaker.py:162]: Jepp, lets do this though once we start logging.

**Pull request merged**